sympy/series/tests/test_limits.py

```diff
@@ -1393,3 +1394,6 @@ def test_issue_25847():
     assert limit(acsch(sin(x)/x), x, 0, '+-') == log(1 + sqrt(2))
     assert limit(acsch(exp(1/x)), x, 0, '+') == 0
     assert limit(acsch(exp(1/x)), x, 0, '-') == oo
+
+def test_issue_26027():
```

Comment:
I think the issue being fixed is `10382` and not `26027`. Make this change and add the function at the appropriate place.